lib/bkblz/v1/get_download_authorization.rb

```diff
@@ -0,0 +1,34 @@
+module Bkblz
+  module V1
+
+    class GetDownloadAuthorizationResponse < Response
+      response_model Model::DownloadAuth
+    end
+
+    class GetDownloadAuthorizationRequest < Request
+
+      response_class GetDownloadAuthorizationResponse
+      url_suffix "/b2api/v1/b2_get_download_authorization"
+      
+      attr_reader :bucket_id, :file_name_prefix, :valid_duration_in_seconds, :content_disposition
+
+      def initialize(bucket_id, file_name_prefix, valid_duration_in_seconds, content_disposition=nil)
+        @bucket_id = bucket_id
+        @file_name_prefix = file_name_prefix
+        @valid_duration_in_seconds = valid_duration_in_seconds
+        @content_disposition = content_disposition
```

Comment:
none of these need to be instance variables. only `@body` does because it's used down in `build_request`